apps/emqx/mix.exs

```diff
@@ -0,0 +1,40 @@
+defmodule EMQX.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :emqx,
+      version: "5.0.0",
+      build_path: "../../_build",
+      config_path: "../../config/config.exs",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+      description: "EMQ X"
+    ]
+  end
+
+  def application do
+    [
+      mod: {:emqx_app, []},
+      extra_applications: [:logger]
+    ]
+  end
+
+  defp deps do
+    [
+      {:gproc, "~> 0.9"},
```

Comment:
we want trust the tags, and pin exact version tags, so we do not have to use the lock file.
this will make it easier to verify integrity between rebar3 and mix when they co-exist.